paper-ieee-trans.tex

```diff
@@ -351,13 +352,13 @@ \section{New trends in software reliability}
 profitable in terms of the evaluation and application of innovative software engineering
 methodologies, the management of new technologies to put those methodologies
 into practice, and the compilation and analysis of the appropriate set of metrics to measure the
-success of the solutions implemented. All the above achievements were with the additional complexity of being framed into
-large collaborative projects with an extensive list of partners.
+quality of the solutions implemented. All the above with the additional complexity of being framed into a
+large collaborative project with an extensive list of partners.
```

Comment:
projects are more than one, we are referring here to the past projects reviewed

Reply:
also think it should be
complexity of being framed into large collaborative projects with an

Reply:
ok

Reply:
ok, I thought it was about one project.